### DISCUSSION TITLE
RFC: New name for common-web-frontend

### DISCUSSION THREAD

**Discussion by oliverschuerch** [swisspost/design-system#304] [Polls]
Most of us have probably thought that the name **common-web-frontend** is not very appropriate.
We are currently thinking about what the framework could be called in the future and want to hear your opinion.

The name should consider the following aspects:
- Affiliation to SwissPost must be clear.
- The name must immediately make clear what it is about (e.g. Framework, DesignSystem, etc.).
- The name should consist of 1-4 words.
- The name should be easily abbreviated (e.g. lorem ipsum => li), but without being too close to another meaning (e.g. political meaning, etc.).
- The name may disrespect some of the previous points, if it is an imaginary name, whose meaning fits the SwissPost and the framework.
- The short form does not have to reflect the name completely (e.g. name: Swiss Post Design System, shortform: post)

We've already started to think about this in a small team, but you are welcome to make further suggestions, which we'll include in the survey.

**Comment by oliverschuerch** [comment]
The decision has been made:
Name: **Swiss Post Design System**
Shortform: **post**

**Comment by gfellerph** [comment]
This change was mainly triggered in an effort to align development resources with the design and the documentation. [Design System](https://en.wikipedia.org/wiki/Design_system) is the officially coined term and thus a perfect way to describe what it is by simply naming it. Swiss Post gives the package the correct scope.

Other namings were also aligned with this descision, for example package names:

Pattern: [company scope]/[project]-[target platform]-[package name]-[modifier/s] (the web platform is implied as default platform).

@swisspost/design-system-components
@swisspost/design-system-components-angular
@swisspost/design-system-components-vue
@swisspost/design-system-header
@swisspost/design-system-intranet-header
@swisspost/design-system-migrations
@swisspost/design-system-styles

The short form post was chosen because of its superior recognizability over any abbreviation. This will be used to prefix CSS variables (#358) and web components (<post-accordion>).
